ert_shared/hook_implementations/site_config.py

```diff
@@ -1,12 +1,17 @@
+from os import uname
+
 from ert_shared.plugins.plugin_manager import hook_implementation
 from ert_shared.plugins.plugin_response import plugin_response
 
 
+lib_extension = "dylib" if uname()[0] == "Darwin" else "so"
```

Comment:
Could we make this variable "private" maybe? Or maybe define it in the function even?